src/Jackett.Common/Definitions/torrentdownload.yml

```diff
@@ -8,7 +8,7 @@
   followredirect: true
   links:
     - https://www.torrentdownload.info/
-    - https://torrentdownload.unblockit.pro/
+    - https://torrentdownload.unblockit.one/
     - https://torrentdownload.black-mirror.xyz/
```

Comment:
you've missed legacylink for this one ;-)

Reply:
There's always one 😋